### PR TITLE
[OTA-912] User notification

### PR DESCRIPTION
**Pull request by JustynaKK**

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/pages/Devices.jsx [hunk 1]
```diff
@@ -16,7 +14,7 @@ const title = "Devices";
 class Devices extends Component {
 
     componentWillMount() {
-        const { devicesStore, groupsStore } = this.props.stores;
+          const { devicesStore, groupsStore } = this.props.stores;
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/pages/Devices.jsx]: Whitespace.

File: ota-plus-web/app/reactapp/src/stores/FeaturesStore.js [hunk 2]
```diff
@@ -1,22 +1,28 @@
 import { observable } from 'mobx';
 import axios from 'axios';
 import {
-    API_FEATURES_FETCH
+    API_FEATURES_FETCH,
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/FeaturesStore.js]: Comma.

File: ota-plus-web/app/reactapp/src/stores/data/newFeatures.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                step: {
+                    id: "smartGroups",
+                    order: 2,
+                    next: "smartFilters",
+                    previous: "introduction",
+                },
+            },
+            smartFilters: {
+                feature: {
+                    title: "More About filter criteria for smart groups",
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/data/newFeatures.js]: "about"

File: ota-plus-web/app/reactapp/src/stores/data/newFeatures.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+                    order: 3,
+                    next: "createUpdate",
+                    previous: "smartGroups",
+                },
+            },
+            createUpdate: {
+                feature: {
+                    title: "Introducing the “Updates” workflow",
+                    message: {
+                        __html: "<p>You use this workflow to put your update tasks into groups called “Updates”. For example, you suppose want to update some voice control software from version 1 to 2. You also want to update the parking assistance software from version 10 to 11. You can group these two tasks into one “Update” call it something like “IVI Updates Sept 2018”.</p>" +
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/data/newFeatures.js]: "suppose you want"

Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/data/newFeatures.js]: "into one Update and call it"

File: ota-plus-web/app/reactapp/src/config.js [hunk 5]
```diff
@@ -18,7 +19,7 @@ export const API_USER_CHANGE_PASSWORD = '/user/change_password';
 export const API_USER_ACTIVE_DEVICE_COUNT = '/api/v1/active_device_count';
 export const API_USER_DEVICES_SEEN = '/api/v1/auditor/devices_seen_in';
 
-export const API_FEATURES_FETCH = '/api/v1/features';
+export const API_FEATURES_FETCH = '/api/v1/popover';
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/config.js]: What is this endpoint?

**Comment by jerrytrieu** [comment]
@JustynaKK can you comment any manual testing you did on any part of the UI that may be affected by this change?

**Comment by JustynaKK** [comment]
> @JustynaKK can you comment any manual testing you did on any part of the UI that may be affected by this change?

I have performed manual testing and this will not affect any part of the the UI.

**Comment by BeneBuz** [comment]
i have tested this in the UI review 
https://saeljira.it.here.com/browse/OTA-1109

**Pull request merged**